doc/ref_model/chapters/chapter09.md

```diff
@@ -306,3 +318,131 @@ Topics include:
+
+As the transition towards cloud environments takes place simultaneously with the introduction of DevOps mindset, conventional monitoring approach becomes suboptimal. Cloud environments allow greater flexibility as the microservice architecture is embraced to bring improvements in operability, therefore the automation can be utilized to much higher extent than ever before. Automation in telecom networks usually supposes actions based on decisions derived from system output data (system observation). In order to derive useful decisions, data with rich context are inevitable. Obviously, conventional monitoring approach has to be improved in order to retrieve sufficient data, not only from the wider context, but also without delays - as soon as data is produced or available. The new enhanced approach was introduced as a concept of observability, borrowed from the control theory which states that it is possible to make conclusions about the system internal state based on the external outputs.
+
+This requires the collection of alarms and telemetry data from the physical layer (wires), the cloud infrastructure up to the network, application and services virtualized functions (VNF) running on top of Cloud Infrastructure, typically isolated by tenants.
+
+Long term trending data are essential for capacity planning purposes and typically collected, aggregated and kept over the full lifespan. To keep the amount of data collected manageable, automatic data reduction algorithms are typically used, e.g. by merging data points from the smallest intervals to more granular intervals.
+
+A cloud typically consists of one or more regional data centers, central offices, and edge sites. These are managed from redundant central management sites, each hosted in their own data centers.
```

Comment:
I think it would be better to say 'telecom infrastructure' here and even that might be forward looking as I don't think today we can generally say that majority of telecom clouds have edge sites. Cloud itself doesn't imply a distributed system and even though this document is targeted for a mainly telco audience I think it's important to distinguish.

Reply:
Good suggestion - fixed the text